assets/js/src/core/modules/element/element-api-slice-enhanced.ts

```diff
@@ -13,5 +13,42 @@
 
 import { type AssetPermissions } from '@Pimcore/modules/asset/asset-api-slice.gen'
 import { type DataObjectPermissions } from '@Pimcore/modules/data-object/data-object-api-slice.gen'
+import { api as baseApi } from './element-api-slice.gen'
+import { invalidatingTags, type Tag, tagNames } from '@Pimcore/app/api/pimcore/tags'
 
 export type ElementPermissions = AssetPermissions & DataObjectPermissions
+
+const api = baseApi.enhanceEndpoints({
+  addTagTypes: [tagNames.ASSET_DETAIL],
+  endpoints: {
+    elementFolderCreate: {
+      invalidatesTags: (result, error, args) => {
+        const tags: Tag[] = []
+
+        if (args.elementType === 'asset') {
+          tags.push(...invalidatingTags.ASSET_DETAIL_ID(args.parentId))
```

Comment:
We cannot just invalidate the element detail tags as this would also invalidate it for the opened element tab. We need to find a different solution for this (based on the related tree tags?)

Reply:
as discussed I will double check the `*_TREE_ID` tag